Update README.md to include relevant links

MitoG opened the pull request:
Added relevant links to the official Repo, Website, Documentation and Discord.

MitoG committed:
Update README.md to include relevant links

Added relevant links to the official Repo, Website, Documentation and Discord.

moiph commented on README.md:
```diff
@@ -1,2 +1,21 @@
+
+----
+
+### Articles
```
I'd rather leave out the articles section and just leave the top level Docs link above

MitoG replied:
that's fine, was just an idea.

moiph replied:
Once you update to remove the articles section I'll get the rest merged

MitoG replied:
README has been updated, the links have been removed.

MitoG committed:
Update README.md

Removed the unwanted links to the articles as per moiph's request 
https://github.com/moiph/ub3r-b0t-docs/pull/1#discussion_r236577050
